Makefile

```
@@ -223,3 +228,26 @@ mv $(1) $(1)-$(3) ;\
 } ;\
 ln -sf $(1)-$(3) $(1)
 endef
+
+.PHONY: kueue
+kueue:
+	kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/$(KUEUE_VERSION)/manifests.yaml -f hack/kueue-config.yaml
+	kubectl rollout status deployment/kueue-controller-manager -n kueue-system --timeout 300s
+
+.PHONY: tekton
+tekton:
+	kubectl apply --server-side -f https://storage.googleapis.com/tekton-releases/pipeline/previous/$(TEKTON_VERSION)/release.yaml
+	kubectl wait --for=condition=Available deployment --all -n tekton-pipelines --timeout=300s
+
+.PHONY: cert-manager
+cert-manager:
+	kubectl apply --server-side -f https://github.com/cert-manager/cert-manager/releases/download/$(CERT_MANAGER_VERSION)/cert-manager.yaml
+	kubectl wait --for=condition=Available deployment --all -n cert-manager --timeout=300s
+
+.PHONY: cert-manager-undeploy
+cert-manager-undeploy:
+	kubectl delete -f https://github.com/cert-manager/cert-manager/releases/download/$(CERT_MANAGER_VERSION)/cert-manager.yaml
+
+.PHONY: load-image
+load-image: docker-build
+	dir=$$(mktemp -d) && podman save $(IMG) -o $${dir}/tekton-kueue.tar && kind load image-archive -n $(KIND_CLUSTER)  $${dir}/tekton-kueue.tar
```

if everything goes fine, shouldn't we delete the folder too?

```suggestion
	dir=$$(mktemp -d) && podman save $(IMG) -o $${dir}/tekton-kueue.tar && kind load image-archive -n $(KIND_CLUSTER)  $${dir}/tekton-kueue.tar && rm -r "${{dir}}"
```

---

done